packages/browser/src/profiling/integration.ts

```diff
@@ -29,6 +30,7 @@ export class BrowserProfilingIntegration implements Integration {
 
   public readonly name: string;
 
+  /** @deprecated This is never set. */
   public getCurrentHub?: () => Hub;
```

Comment:
I think we can just remove this tbh, nobody should be relying on it.

Reply:
yeah, I'll remove it in a next step where I rewrite this to functional style!